higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -527,7 +530,13 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
                   excerpt.innerHTML =  '<h5>'+entryInfo.entryName + ' '+ entryInfo.entrySurname + ' | ' + entryInfo.entryAge() + ' | ' + entryInfo.entryAddress+'</h5>';
                 }
                 else if(collapsiblesToggle === 'step_2') {
-                  excerpt.innerHTML = '<h5>'+ entryInfo.entryUni() +' | '+entryInfo.entryCampus()+'</h5>';
+                  if(entryInfo.entryUni() === 'Other' || entryInfo.entryCampus() === 'Other') {
+                    excerpt.innerHTML = '<h5>'+ entryInfo.entryUniOther +' | '+ entryInfo.entryCampusOther +'</h5>';
+                    console.log('True');
```

Comment:
Are we keeping these console logs in to test in QA, and removing before production?

Reply:
Good catch! Thanks @rudigiesler. Removing...